app/Css/Post.hs

```diff
@@ -83,34 +105,46 @@ postCSS = do
         semiVisible
         ":hover" & do
             fullyVisible
+    ".code:before" ? do
+            "content" -: "'✔'"
```

Comment:
I got an encoding issue here. Safer to use the CSS code: `"content" -: "'\2713'"` (and below)